src/Dodo.HttpClient.ResiliencePolicies.Tests/DSL/HttpClientWrapperBuilder.cs

```diff
@@ -12,11 +12,12 @@ namespace Dodo.HttpClientResiliencePolicies.Tests.DSL
 	public sealed class HttpClientWrapperBuilder
 	{
 		private const string ClientName = "TestClient";
+		private Uri _uri = new Uri("http://localhost");
```

Comment:
_uri could be marked as readonly. Also we could just inline this.

Reply:
I fix it.